airbyte-webapp/src/packages/cloud/services/cloudApi/useSpeakeasyRedirect.ts

```diff
@@ -0,0 +1,13 @@
+import { getSpeakeasyCallbackUrl } from "packages/cloud/lib/domain/cloudApi";
+import { useSuspenseQuery } from "services/connector/useSuspenseQuery";
+import { useDefaultRequestMiddlewares } from "services/useDefaultRequestMiddlewares";
+
+const SPEAKEASY_QUERY_KEY = "speakeasy-redirect";
+
+export const useSpeakeasyRedirect = () => {
+  const config = { apiUrl: "/cloud_api" };
```

Comment:
This is actually not the same `cloudApiUrl` from `useConfig()` - it's a new API URL that is being introduced by the platform enablement team. Agreed that hardcoding here is not great, but it's the only place it's used currently. Happy to clean this up Monday and add a new config property once we decide on a naming scheme.

Reply:
let's work on Monday together on that. We should make sure if this is a new endpoint, that we add it properly to the env files etc so it can be env independant. We should wait for the BE to actually be deployed for this, so we can alidate that this is working properly.